tests/default/data/doctrine-dbal.php

```diff
@@ -218,4 +219,49 @@ public function dateParameter(Connection $conn)
         $fetchResult = $conn->fetchOne($query, [date('Y-m-d', strtotime('-3hour'))]);
         assertType('int|false', $fetchResult);
     }
+
+    public function customTypeParameters(Connection $conn, int $adaid)
+    {
+        $stmt = $conn->prepare('SELECT count(*) AS c FROM typemix WHERE c_datetime=:last_dt');
+        $result = $stmt->execute(['dt' => new \DateTime()], ['dt' => Types::DATETIME_MUTABLE]);
```

Comment:
does doctrine allow to pass in a `DateTime` object without also giving a parameter-type as a 2nd arg?

or should phpstan-dba error when arg1 is DateTime and arg2 is missing or not equal to `Types::DATETIME_MUTABLE`?

Reply:
I am not sure if it lets you do it without the param type, but I'd think not.